src/dbwrapper.cpp

```diff
@@ -89,6 +89,7 @@ static leveldb::Options GetOptions(size_t nCacheSize)
 }
 
 CDBWrapper::CDBWrapper(const fs::path& path, size_t nCacheSize, bool fMemory, bool fWipe, bool obfuscate)
+    :m_name(fs::basename(path))
```

Comment:
Nit, space after `:`.

Reply:
BTW, could use `path` in the debug instead?
 - relevant considering #11687
 - CDBWrapper constructor also logs path.

Reply:
Added space. I kind of like the short path in the logs because it makes the lines more readable, what if I did the following:
 * Add new name parameter to constructor
 * Chainstate/blocks databases will pass in "chainstate" "blocks" etc. to explicitly name the object
 * If an explicit name is omitted the path will be logged, and this pattern could be used in #11687 since wallet files are not global like the other databases.